aws/kinesis-stream/main.tf

```
@@ -0,0 +1,86 @@
+locals {
+  provisioned_mode = var.stream_mode == "PROVISIONED"
```

```suggestion
  is_provisioned_mode = var.stream_mode == "PROVISIONED"
```

Up to you on this one - I generally try to name boolean variables with a name that implies they are a boolean.